core/src/main/java/io/grpc/internal/ManagedChannelImplBuilder.java

```diff
@@ -878,13 +878,14 @@ public String getServiceAuthority() {
 
         @Override
         public void start(Listener2 listener) {
-          listener.onResult2(
-              ResolutionResult.newBuilder()
-                  .setAddressesOrError(
-                      StatusOr.fromValue(
-                          Collections.singletonList(new EquivalentAddressGroup(address))))
-                  .setAttributes(Attributes.EMPTY)
-                  .build());
+          args.getSynchronizationContext().execute(() ->
```

Comment:
I don't see why this was added. [`start()` should be called from the synchronization context.](https://grpc.github.io/grpc-java/javadoc/io/grpc/NameResolver.Args.html#getSynchronizationContext())

Reply:
Removed.